wapi/util.py

```diff
@@ -37,8 +38,11 @@
     'MIN5': '5T',
     'MIN': 'T',
 }
-# Mapping from Pandas to TS is built from map above
-_PANDAS_FREQ_TABLE = {}
+# Mapping from Pandas to TS is built from map above, with some additions
+_PANDAS_FREQ_TABLE = {
+    'AS-JAN': 'Y',
+    'QS-JAN': 'Q',
+}
```

Comment:
What's the reason we only want these specific two?

Reply:
No particular reason, it's those I've actually seen in curves created by Pandas so far.  Any more we should add?

Reply:
I'm not sure – a lot of the possibilities seem very unlikely to show up ever. We already cover `W-MON`, but `W-SUN` is the actual pandas `W` – might have some suprising results if creating a TS from a pandas dataframe without using `wapi.util.from_pandas`?

The complete list (for reference): http://pandas-docs.github.io/pandas-docs-travis/user_guide/timeseries.html?highlight=offset%20aliases#anchored-offsets

Reply:
We can't add those that mean something else, then it's better for things to crash.  So `W-SUN` is not wapi-compatible and the user needs to fix it...

Those in there right now are exactly equivalent to what wapi uses, so the data becomes correct...